### ISSUE TITLE
Wrong Output of Brats 3D Segmentation

### ISSUE THREAD

**Issue by vkingoff** [Project-MONAI/tutorials#171]
I ran your brats_segmentation_3d.ipynb and got results which are different from yours. 
The versions of my environment are listed below and i attach the picture with final results.
![image_2021-04-03_23-57-25](https://user-images.githubusercontent.com/34441268/113491300-65ff2980-94d8-11eb-9f3d-10cfa32f4343.png)
```
MONAI version: 0.5.dev2113
Numpy version: 1.19.1
Pytorch version: 1.7.0a0+8deb4fe
MONAI flags: HAS_EXT = False, USE_COMPILED = False
MONAI rev id: b3cc668c924b2a7e6b822416f4d920a098c27704

Optional dependencies:
Pytorch Ignite version: NOT INSTALLED or UNKNOWN VERSION.
Nibabel version: 3.2.1
scikit-image version: 0.17.2
Pillow version: 8.1.2
Tensorboard version: 1.15.0+nv
gdown version: NOT INSTALLED or UNKNOWN VERSION.
TorchVision version: 0.8.0a0
ITK version: NOT INSTALLED or UNKNOWN VERSION.
tqdm version: 4.59.0
lmdb version: 1.0.0
psutil version: 5.7.2
```

**Comment by rijobro** [comment]
Thanks for raising this. It looks like during the plotting we have the line `val_output = model(val_input)`, which should probably be `val_output = post_trans(model(val_input))`. 

Do you think you could try modifying it to this and seeing if this gives expected results?

**Comment by vkingoff** [comment]
Maybe I will have a chance to check it later. But for now I was able to solve it by the next change.
Your string:
plt.imshow(val_output[0, i, :, :, 20].detach().cpu())
My string:
plt.imshow(torch.sigmoid(val_output[0, i, :, :, 20]).detach().cpu()>0.5)

**Comment by rijobro** [comment]
That will have the same effect as `post_trans`, glad it worked for you!
